lib/OpenQA/Resource/Locks.pm

```diff
@@ -19,8 +19,7 @@ my %final_states = map { $_ => 1 } OpenQA::Jobs::Constants::NOT_OK_RESULTS();
 # So we have to specify, which child job is supposed to create the lock
 # and watch its state.
 #
-sub _get_lock {
-    my ($name, $jobid, $where) = @_;
+sub _get_lock ($name = undef, $jobid = undef, $where = undef) {
```

Comment:
I think `_get_lock` is always called with 3 arguments, so
```suggestion
sub _get_lock ($name, $jobid, $where) {
```

Reply:
The same counts probably for `unlock` as well.

Reply:
ok, removed the `= undef` and also the obsolete safety checks for variables after that then